src/common/DeveloperConfig.ts

```diff
@@ -36,5 +37,9 @@ export function getCustomAppTileComponent(
   appTileScreens?: AppTileScreens,
   appTile?: AppTile,
 ) {
-  return !!appTile?.source.url && appTileScreens?.[appTile.source.url];
+  return (
+    !!appTile?.source.url &&
+    (appTileScreens?.[appTile.source.url] ||
+      appTile.source.url.startsWith(getBundleId()))
+  );
```

Comment:
So if we have configured a screen for it or if it matches our bundle id, that seems like the later would only apply in a case where we have a misconfiguration. i.e. we have not provided the screen in our developer config but the url matches our bundleId.